package.json

```diff
@@ -54,18 +54,19 @@
   "devDependencies": {
     "asini-changelog": "^1.3.0",
     "babel-cli": "^6.7.5",
-    "babel-eslint": "^6.0.2",
+    "babel-eslint": "^7.1.1",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "babel-preset-es2015": "^6.6.0",
     "babel-register": "^6.7.2",
     "cross-env": "^1.0.8",
-    "eslint": "^2.3.0",
-    "eslint-config-babel": "^1.0.1",
-    "eslint-plugin-babel": "^3.3.0",
+    "eslint": "^3.10.2",
+    "eslint-config-babel": "^3.0.0",
+    "eslint-plugin-babel": "^4.0.0",
     "eslint-plugin-flow-vars": "^0.5.0",
```

Comment:
We can remove `eslint-plugin-flow-vars`, too, as it is no longer a peer dependency of `eslint-config-babel`.